src/anim/anim.js

```diff
@@ -694,17 +694,25 @@ Object.assign(pc, function () {
                 if (!object ||
                     !object.model ||
                     !object.model.model ||
-                    !object.model.model.morphInstances ||
-                    !object.model.model.morphInstances[0]) {
+                    !object.model.model.morphInstances) {
+                    return null;
+                }
+                var morphInstance;
+                for (var i = 0; i < object.model.model.meshInstances.length; i++) {
+                    if (object.model.model.meshInstances[i].node.name === node.name) {
+                        morphInstance = object.model.model.meshInstances[i].morphInstance;
```

Comment:
These 3 lines could probably be tightened up a bit performance wise. (e.g. `object.model.model` used 3 times...loop counter calculated every iteration, etc).

Reply:
Sure, cleaned this up a bit!